docs/resources/azure_china_subscription.md

```diff
@@ -71,5 +71,7 @@ Required:
 Import is supported using the following syntax:
 
 ```shell
+# Copyright 2022 Nordcloud Oy or its affiliates. All Rights Reserved.
```

Comment:
Do we need this?

Reply:
It was added from the `make generateDoc` execution